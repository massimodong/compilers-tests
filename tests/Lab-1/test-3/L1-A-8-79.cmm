struct PuUBKVy8MJjQCOJ38X ;
int ;
struct {
  int F__brzlZJIJoThZdV ;
}MgysMk6w56CmhrYzVI ( ) {
  float bomF , fX75SpMeROG [ 937188876 ] = X1UlO ;
  if ( agkUg3oPl )   if ( qjllvQ5D4 )   return p3kdAk 
    ;
else   while ( HV )   zolgUtogd4ckFNe = W7UTxXu6CNqV 
    <= j4QB - - p2j [ bt2 = mivvUJi9E8BTYPvSDE || q27RA_R4ZoOifaOWuUk 
    && nungbkHpX > QCKWISeSEfAjoLEdBv7 * Dp ( n , hc && 
    uZva9Tsg2A71_ < krtiKxfL_66UA9PUdu + ! ( occW = Eo7ffRIoFUKa 
    == Y - xPoDn0DclCHL ( WnQi1qQezCGufLni >= d4 - h1prBqeiTh5ZpskZl 
    ( Jj1 != DHhXL ) || 107221580.1673858711 ) / gbkyo 
    ) . J / - 784990496 ) ] / ! - reC5njajiD_wbaVBxwJN 
    ( ) ;



}
int r_K1XXsyZrjFn_ 
;
int zQPta19qNzBPgsbPw ( int b7q4 , int U1dC3o6EFhYt4 
  ) {
}
struct _dtTXaiv_CT ;
struct W79vjtud8rpZFjf1 ;
int JZA1g5_LGq8TxEsiXqhn ( ) {
  yS6FEv51WqMsHlD ;
{
  }}
int ;
int x1xWVV5qkdrgSkosOx , kjSePyTKnrRFAMdvAp2 

;
struct JDYD03wmmpt {
  int r2j2OV1jwu3NJQUvvx ;
};
int ;
int ;
int ;
int ;
int ;
int ;
int w7axW9K5LaxULb7P ( ) {
  if ( GP )   TjyM5J_zb0s ;
else   if ( HgW1QnC )   Y0G ;


  H9Jv3uvToOkQvh ;
}/ 
int ;
int ;
int ;
int vxJ3bSpzGx_GtfK63 ( ) {
{
    if ( PseOY7ySo480 )     ElwRlwjkPI9o ;
else     M5O5VrxJMtOMHRbMbV ;

    if ( nAVWWVpeJ9dhUxV1X )     aEw_YgpjylRylx ;
else     JnTuiy9X34ze56u ;

  }}
[ 