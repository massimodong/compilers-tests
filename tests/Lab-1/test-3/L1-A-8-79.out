Error Type B at Line 51
Error Type B at Line 64
